FittedSheetsPod/SheetSize.swift

```diff
@@ -14,6 +14,7 @@ public enum SheetSize: Equatable {
     case fixed(CGFloat)
     case fullscreen
     case percent(Float)
+    case margin(CGFloat)
```

Comment:
This should probably be `.fullscreenWithTopMargin(CGFloat)`, to indicate what the base size is. `.margin(CGFloat)` doesn't clearly indicate what it will be a margin on.